zq2/docs/sdk.md

```diff
@@ -1,53 +1,44 @@
-- [otterscan](https://github.com/Zilliqa/otterscan) - to add the ability to understand Zilliqa native transactions.
-
-## Sample contracts
+- [Rust](https://crates.io/crates/zilliqa-rs)
+- [Golang](https://github.com/Zilliqa/gozilliqa-sdk)
+- [Java](https://github.com/Zilliqa/zilliqa-developer/tree/main/products/laksaj)
+- [Javascript/typescript](https://www.npmjs.com/package/@zilliqa-js/zilliqa)
+- [Python](https://github.com/zilliqa/pyzil)
 
-- [ZRC](https://github.com/Zilliqa/zrc) contains reference contracts and standards for Zilliqa native contracts.
-- [zilliqa-developer/contracts](https://github.com/Zilliqa/zilliqa-developer/tree/main/contracts) contains more extensive/complex contracts.
+We also continue to maintain tools such as:
 
-In particular, `zilliqa-developer` contains contracts for vesting and burning tokens, and for generating contracts that expose native Zilliqa functionality to EVM (eg. for generating an ERC-20 compatible token contract that manipulates an underlying ZRC-2 asset).
+- [Scilla hardhat plugin](https://github.com/Zilliqa/hardhat-scilla-plugin) - to support deploying and testing Scilla contracts
+- [ethers.js fork](https://github.com/Zilliqa/ethers.js) - to support the non-canonical signatures of non-EVM Zilliqa transactions (Schnorr signatures)
+- [otterscan fork](https://github.com/Zilliqa/otterscan) - to add the ability to understand non-EVM (Scilla) transactions
 
-## Zilliqa source
 
-We provide a number of repositories as open source:
+### Open-source repositories
+We provide a number of open-source repositories relevant to Zilliqa 2.0:
 
 - [zq2](https://github.com/Zilliqa/zq2) - the Zilliqa 2 source code itself.
-- [zilliqa-developer](https://github.com/Zilliqa/zilliqa-developer) - this contains source for, among other things
-  - [eth-spout](https://github.com/Zilliqa/zilliqa-developer/tree/main/products/eth-spout) a simple faucet.
-  - [xbridge](https://github.com/Zilliqa/zilliqa-developer/tree/main/products/bridge) a cross-chain contract bridge.
-  - [multisig](https://github.com/Zilliqa/zilliqa-developer/tree/main/products/multisig) - a scilla multisig implementation
-  - [neo-savant](https://github.com/Zilliqa/zilliqa-developer/tree/main/products/neo-savant) - the Scilla IDE
-  - [pdt](https://github.com/Zilliqa/zilliqa-developer/tree/main/products/pdt) - a utility which turns the state of the blockchain into a BigQuery dataset or PostgreSQL database.
-  - [developer-portal-zq2](https://github.com/Zilliqa/zilliqa-developer/tree/main/products/developer-portal-zq2) - this developer portal
+- [eth-spout](https://github.com/Zilliqa/zilliqa-developer/tree/main/products/eth-spout) - a simple faucet.
+- [neo-savant](https://github.com/Zilliqa/zilliqa-developer/tree/main/products/neo-savant) - the Scilla IDE
+- [developer-portal-zq2](#) - this developer portal
```

Comment:
I think: 
- the developer portal link ought to be to this git repo (admittedly not the link inthe docs!)
- what did multisig do wrong? (it should work as long as scilla does)
-

Reply:
https://github.com/Zilliqa/dev-portal/issues/23